etc/ATSCam/primary_hdu.header

```diff
@@ -5,35 +5,52 @@ EXTEND  =                    T / Extensions are permitted
 ORIGIN  = 'LSST DM Header Service'/ FITS file originator
 DATE    =                      / Creation Date and Time of File
 DATE-OBS=                      / Date of the observation (image acquisition)
+DATE-END=                      / end date of the observation
 MJD     =                      / Modified Julian Date that the file was written
 MJD-OBS =                      / Modified Julian Date of observation
 OBS-LONG=           -70.749417 / [deg] Observatory east longitude
 OBS-LAT =           -30.244639 / [deg] Observatory latitude
 OBS-ELEV=               2663.0 / [m] Observatory elevation
+OBSGEO-X=           1818938.94 / [m] X-axis Geocentric coordinate
+OBSGEO-Y=          -5208470.95 / [m] Y-axis Geocentric coordinate
+OBSGEO-Z=          -3195172.08 / [m] Z-axis Geocentric coordinate
 OBS-NITE=                      / Night of the observation
+OBS-ID  =                      / ImageName from Camera StarIntergration
 TELESCOP= 'LSST AuxTelescope'  / Telescope name
 INSTRUME= 'LATISS'             / Instrument used to obtain these data
-RATEL   =                      / Telescope RA of observation (hr)
-DECTEL  =                      / Telescope DEC of observation (hr)
-RA      =                      / RA of Target (hr)
-DEC     =                      / DEC of Target (deg)
-ROTANGLE=                      / Rotation angle (deg)
-HA      =                      / [HH:MM:SS] Telescope hour angle
-ZD      =                      / [deg] Telescope zenith distance
-AZ      =                      / [deg] Telescope azimuth angle
-AIRMASS =                      / Airmass
-PROGRAM =                      / Current observing orogram
+TELCODE = 'AT'                 / The code for the telecope
+CONTRLLR=                      / The controller (e.g. O for OCS, C for CCS)
+DAYOBS  =                      / The observation day as defined in the image name
+SEQNUM  =                      / The sequence number from the image name
+IMAGETAG=                      / DAQ Image id
+SEQNAME =                      / SequenceName from Camera StartIntergration
+REBNAME =                      / Name of the REB
+IMGTYPE =                      / BIAS, DARK, FLAT, OBJECT
+RATEL   =                      / Telescope RA of observation
+DECTEL  =                      / Telescope DEC of observation
```

Comment:
Are these in degrees? Also, if these are ICRS then say that in the comment or use RADESYS header.  It could be that these are the apparent RA/Dec rather than the tracking RA/Dec. Which is it?

Reply:
Unfortunately, the units and the source of these are yet undefined.